src/Analyzers/CSharp/CodeFixes/NamespaceMatchFolder/CSharpChangeNamespaceToMatchFolderCodeFixProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeFixes.NamespaceMatchFolder;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.NamespaceMatchFolder
```

Comment:
```suggestion
namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.UpdateNamespaceToMatchFolder
```

?

Reply:
or: Microsoft.CodeAnalysis.CSharp.CodeFixes.ChangeNamespaceToMatchFolder

we keep our analyzer/fixer folders in sync with the names of hte analyzer/fixers (ignoring things like the alnguage name, and teh DiagnosticAnalayzer/Provider suffixes.)

Reply:
In this case, when I add an "UpdateFolderToMatchNamespace" would it go in the same folder or not?

Reply:
offline discussion.